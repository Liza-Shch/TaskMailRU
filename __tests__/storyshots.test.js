import initStoryshots from '@storybook/addon-storyshots';
const path = require('path');

initStoryshots({
    framework: 'html',
    config: ({ configure }) =>
      configure(() => {
        require(path.resolve(__dirname, '..', 'public/stories/NumberVerify.story.js'));
      }, module),
  });