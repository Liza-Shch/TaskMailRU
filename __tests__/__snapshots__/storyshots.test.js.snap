// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots NumberVerify error 1`] = `
<div
  class="number-verify-container"
>
  <div
    class="number-verify-number"
  >
    +
    <div
      class="number-verify-number__block"
    />
    (
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    )
    <div
      class="number-verify-number__block"
    />
    <input
      class="number-verify-number__input number-verify-number__input_error"
      placeholder="_"
    />
    <input
      class="number-verify-number__input number-verify-number__input_error"
      placeholder="_"
    />
    -
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    -
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
  </div>
  <div
    class="number-verify-msg number-verify-container__msg number-verify-msg_error"
  >
    Неверный номер, попробуйте еще раз
  </div>
</div>
`;

exports[`Storyshots NumberVerify normal 1`] = `
<div
  class="number-verify-container"
>
  <div
    class="number-verify-number"
  >
    +
    <div
      class="number-verify-number__block"
    />
    (
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    )
    <div
      class="number-verify-number__block"
    />
    <input
      class="number-verify-number__input"
      placeholder="_"
    />
    <input
      class="number-verify-number__input"
      placeholder="_"
    />
    -
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
    -
    <div
      class="number-verify-number__block"
    />
    <div
      class="number-verify-number__block"
    />
  </div>
</div>
`;
